modules/comet_rds/main.tf

```diff
@@ -99,6 +99,16 @@ resource "aws_rds_cluster_parameter_group" "cometml-cluster-pg" {
     name         = "thread_stack"
     value        = "2000000"
   }
+  parameter {
+    apply_method = "pending-reboot"
+    name         = "group_concat_max_len"
+    value        = "1000000"
+  }
+  parameter {
+    apply_method = "pending-reboot"
```

Comment:
I don't think this setting required a reboot

Reply:
None of these parameters in the way they're set here require a reboot after creation, but some of them will throw errors and block the apply if set to 'immediate'. So for usability they are set by default to 'pending-reboot'